extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -276,6 +290,16 @@ export class LiveListManager {
     this.liveLists_[id] = liveList;
     this.intervals_.push(liveList.getInterval());
 
+    // Origin Trial for cache busting requests for `amp-live-list`.
+    installOriginExperimentsForDoc(this.ampdoc);
+    originExperimentsForDoc(liveList.element)
+      .getExperiments()
+      .then(
+        trials =>
+          (this.enrolledInAppendRandomExperiment_ =
+            trials && trials.includes('amp-live-list-random'))
+      );
```

Comment:
Consider making `enrolledInAppendRandomExperiment` a tri-bool so this only happens once:

```js
if (this.enrolledInAppendRandomExperiment_ === undefined) {
  ...
}
```

Reply:
Implemented this change.